pipelines/azure-pipelines.yml

```diff
@@ -9,16 +9,25 @@ steps:
     displayName: 'Use NuGet 4.7.1'
     inputs:
       versionSpec: '4.7.1'
+
   - task: NuGetCommand@2
     displayName: 'Restore NuGet Packages'
     inputs:
       command: 'restore'
       restoreSolution: '**/*.sln'
       feedsToUse: 'select'
       vstsFeed: 'c5095ddb-6433-4990-a3a5-1df8a78b1302/$(FeedSecret)'
+
   - task: DotNetCoreCLI@2
     displayName: 'Build Function App'
     inputs:
       command: 'build'
       projects: 'src/Dfe.Spi.EntitySquasher/Dfe.Spi.EntitySquasher.FunctionApp/Dfe.Spi.EntitySquasher.FunctionApp.csproj'
-      arguments: '--configuration Release --output $(Build.BinariesDirectory) --no-restore'
\ No newline at end of file
+      arguments: '--configuration $(BuildConfiguration) --output $(Build.BinariesDirectory) --no-restore'
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Run Unit Tests'
+    inputs:
+      command: test
+      projects: 'src/**/*.UnitTests.csproj'
+      arguments: '--configuration $(BuildConfiguration) --no-restore'
```

Comment:
You are missing the publish step, so no artifact is being published for the build

Reply:
This is being done in the previous step, at least, the preparing of the artfiact. I'm not publishing it yet, as I'm waiting for the environments still. I was just going to do it when I started to wire things up. I could do it now though, if you like? 🤷‍♂️